src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="get"
+    class="form form__big"
+    data-qa="big"
+    >
```

Comment:
Still not fixed from the previous review
```suggestion
    <form
      action="#"
      method="get"
      class="form form__big"
      data-qa="big"
    >
```